config/sync/views.view.manage_collection_solutions.yml

```diff
@@ -0,0 +1,532 @@
+uuid: 7c42c34d-2943-430f-80ef-777fbba1d8da
+langcode: en
+status: true
+dependencies:
+  config:
+    - rdf_entity.rdfentity.collection
+    - search_api.index.published
+  module:
+    - joinup_group
+    - search_api
+    - solution
+id: manage_collection_solutions
+label: 'Manage collection solutions'
+module: views
+description: 'Allows management of solutions within a collection.'
+tag: ''
+base_table: search_api_index_published
+base_field: search_api_id
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: manage_solutions
+        options: {  }
+      cache:
+        type: tag
+        options: {  }
+      query:
+        type: views_query
+        options:
+          bypass_access: false
+          skip_access: false
+          preserve_facet_query_args: false
+      exposed_form:
+        type: basic
+        options:
+          submit_button: Apply
+          reset_button: false
+          reset_button_label: Reset
+          exposed_sorts_label: 'Sort by'
+          expose_sort_order: true
+          sort_asc_label: Asc
+          sort_desc_label: Desc
+      pager:
+        type: full
+        options:
+          items_per_page: 10
```

Comment:
The reason I went with 10 rows is that I wanted to avoid a batch process when changing the collection. As this is a moderators only task, I wanted to avoid too much effort (it's already enough!). Let's say we can increase to 20 rows (probably 30?) but certainly not 100.

Reply:
Increased to 20 items per page.